test/decorators.py

```diff
@@ -33,11 +33,12 @@
 
 from qiskit.test.testing_options import get_test_options
 from qiskit_ibm import least_busy
-from qiskit_ibm.ibmqfactory import IBMQFactory
+from qiskit_ibm import IBMAccount
 from qiskit_ibm.credentials import (Credentials,
                                     discover_credentials)
-from qiskit_ibm.accountprovider import AccountProvider
-from qiskit_ibm import IBMQ
+from qiskit_ibm.ibm_provider import IBMProvider
+
+ACCOUNT = IBMAccount()
```

Comment:
We should probably get away from using a global variable like this. But that can be a separate PR.

Reply:
Fixed!